obspy/io/nordic/core.py

```diff
@@ -1390,7 +1390,10 @@ def nordpick(event, high_accuracy=True):
                 warnings.warn("Multiple arrivals for pick - only writing one")
             arrival = arrival[0]
             # Extract weight - should be stored as 0-4, or 9 for seisan.
-            weight = _str_conv(int(arrival.time_weight or 0))
```

Comment:
It looks like, because of the change to `_str_conv` this duplicates the logic. It might be best to just make the change to `_str_conv` to reduce duplication.

Reply:
Sorry, I thought I had left this comment long ago but I never hit the right button.  It answers Calum's question below.  @calum, if you prefer that un-specified pick accuracies are tagged with "0" I'll take this out of the code (fixing the _str_conv() bug is much more important)

Wayne